tubelib/node_states.lua

```diff
@@ -175,9 +176,27 @@ function NodeStates:start(pos, meta, called_from_on_timer)
 		if self.formspec_func then
 			meta:set_string("formspec", self.formspec_func(self, pos, meta))
 		end
-		minetest.get_node_timer(pos):start(self.cycle_time)
+		local cycle_time = self.cycle_time
+		if self.first_cycle_time then
+			if meta:get_int("tubelib_first_run") == 1 then
+				meta:set_int("tubelib_first_run", 0)
```

Comment:
I'm not 100% sure setting a metadata prop is the ideal way to do this, let me know

Reply:
I don't see a better, quicker solution either